From fa317885b245fec77000d559533a30747d652aa1 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <josef.schlehofer@nic.cz>
Date: Tue, 18 Sep 2018 14:31:35 +0200
Subject: [PATCH] libsodium: update to version 1.0.16

---
 libs/libsodium/Makefile | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/libs/libsodium/Makefile b/libs/libsodium/Makefile
index dc75b4358..23ca78e47 100644
--- a/libs/libsodium/Makefile
+++ b/libs/libsodium/Makefile
@@ -8,12 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libsodium
-PKG_VERSION:=1.0.10
+PKG_VERSION:=1.0.16
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://download.libsodium.org/libsodium/releases
-PKG_MD5SUM:=ea89dcbbda0b2b6ff6a1c476231870dd
+PKG_SOURCE_URL:=https://download.libsodium.org/libsodium/releases \
+https://github.com/jedisct1/libsodium/releases/download/$(PKG_VERSION)
+PKG_HASH:=eeadc7e1e1bcef09680fb4837d448fbdf57224978f865ac1c16745868fbd0533
 
 PKG_FIXUP:=libtool autoreconf
 PKG_USE_MIPS16:=0
@@ -27,6 +28,7 @@ include $(INCLUDE_DIR)/package.mk
 define Package/libsodium
   SECTION:=libs
   CATEGORY:=Libraries
+	DEPENDS:=+libpthread
   TITLE:=P(ortable|ackageable) NaCl-based crypto library
   URL:=https://github.com/jedisct1/libsodium
   MAINTAINER:=Damiano Renfer <damiano.renfer@gmail.com>
-- 
2.17.1

